client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -645,7 +645,10 @@ const mapStateToProps = (
 		queryArgs = wpcom.addSupportParams( queryArgs );
 	}
 
-	const siteAdminUrl = getSiteAdminUrl( state, siteId, postId ? 'post.php' : 'post-new.php' );
+	const siteAdminUrl =
+		postType === 'site'
```

Comment:
I don't think this should be called `postType` since this an existing WP term which can take a couple of predefined values like `post`, `page`, or CPT name etc. What do you think about about calling it `editorType` instead?

Reply:
That makes sense.  I have been thinking of 'site' as a postType like post or page, but really that doesn't fully make sense as it is the entire site...  I updated this to use `editorType`